### ISSUE TITLE
Editable SVG Component 

### ISSUE THREAD

**Issue by LeBogoo** [motion-canvas/motion-canvas#509]
**Description**
I am loading in an SVG from Adobe Illustrator as an Img component and I am trying to edit the styles of that SVG image.
The problem is: I can't find a way to edit anything inside the Img component.

It would be really helpful, if there would be an SVG component that could be edited.
This way you can easily create graphics and animate them in Motion Canvas.

**Proposed solution**
The SVG component should be able to take in a src (and a ref) prop, wich will be parsed into child elements.
This is how i would image to use it:
```tsx
import svgSource from "../image.svg"

...
const svgRef = createRef<Svg>()
view.add(<Svg src={svgSource} ref={svgRef} />);

for (let child of svgRef().children()) {
    yield* child.scale(2, 1);
}
...
```

**Comment by polRk** [comment]
FYI https://www.npmjs.com/package/vite-plugin-svgr

**Comment by aarthificial** [comment]
@polRk Motion Canvas doesn't use React

Also, closing this issue because an [SVG](https://motioncanvas.io/api/2d/components/SVG) node has been implemented in #763
